ex3/ex3.c

```diff
@@ -9,7 +9,13 @@
 
 int main(void)
 {
-    // Your code here
+    int count = fork();
+    if(count != 0) {
```

Comment:
I'm not sure if this logic guarantees that hello is always printed first. To guarantee it you'd need the wait command. Also in this simple example it's technically fine to use count != 0 to indicate the parent process, but I imagine this could cause some unexpected behavior in a larger application. You may find it more convenient to reserver your if() statement for the child process with the if (count == 0) logic. Remember child creates a copy of the parent process. With this syntax it's almost as if parent is a "copy" of child.